ban/http/api.py

```diff
@@ -391,18 +408,32 @@ def get_version(self, identifier, ref):
     @app.jsonify
     @app.endpoint('/<identifier>/versions/<int:ref>/flag', methods=['POST'])
     def post_version(self, identifier, ref):
-        """Flag a version.
+        """Flag a {resource} version.
 
         parameters:
             - $ref: '#/parameters/identifier'
             - name: ref
               in: path
-              type: string
+              type: integer
               required: true
               description: version reference, either a date or an increment.
+            - name: body
+              in: body
+              type: string
```

Comment:
Now I have a new doubt: does it make sense to have "type: string" with a "schema"? Shouldn't this be "type: object" instead?